### DISCUSSION TITLE
Real-time failure in FAPI: TRYING TO CONNECT COTS UE

### DISCUSSION THREAD

**Discussion by ChiefMeta** [srsran/srsRAN_Project#483] [General Help]
After grading to the latest srsRAN and open5gs. I cannot connect the COTS UE anymore. Can you please help?

i already ran the performance script.

Could not open /sys/fs/cgroup/cgroup.procs for writing. error="Read-only file system"

The PRACH detector will not meet the performance requirements with the configuration {Format 0, ZCZ 0, SCS 1.25kHz, Rx ports 1}.
Lower PHY in quad executor mode.

--== srsRAN gNB (commit 0b2702cca) ==--

Connecting to AMF on 127.0.1.100:38412
Available radio types: uhd.
[INFO] [UHD] linux; GNU C++ version 9.2.1 20200304; Boost_107100; UHD_3.15.0.0-2build5
[INFO] [LOGGING] Fastpath logging disabled at runtime.
No device type given, found device with address 'type=x300,addr=192.168.30.2,fpga=XG,name=,serial=F5FAEF,product=X310'
Making USRP object with args 'type=x300,master_clock_rate=184.32e6,send_frame_size=8000,recv_frame_size=8000'
[INFO] [X300] X300 initialization sequence...
[INFO] [X300] Maximum frame size: 7972 bytes.
[WARNING] [X300] You requested a receive frame size of (8000) but your NIC's max frame size is (7972).Please verify your NIC's MTU setting using 'ip link' or set the recv_frame_size argument appropriately.UHD will use the auto-detected max frame size for this connection.
[WARNING] [X300] You requested a send frame size of (8000) but your NIC's max frame size is (7972).Please verify your NIC's MTU setting using 'ip link' or set the send_frame_size argument appropriately.UHD will use the auto-detected max frame size for this connection.
[WARNING] [X300] For the 192.168.30.2 connection, UHD recommends a receive frame size of at least 8000 for best
performance, but your configuration will only allow 7972.This may negatively impact your maximum achievable sample rate.
Check the MTU on the interface and/or the recv_frame_size argument.
[INFO] [X300] Radio 1x clock: 184.32 MHz
[INFO] [GPS] Found an internal GPSDO: LC_XO, Firmware Rev 0.929a
[INFO] [0/DmaFIFO_0] Initializing block control (NOC ID: 0xF1F0D00000000000)
[INFO] [0/DmaFIFO_0] BIST passed (Throughput: 1310 MB/s)
[INFO] [0/DmaFIFO_0] BIST passed (Throughput: 1313 MB/s)
[INFO] [0/Radio_0] Initializing block control (NOC ID: 0x12AD100000000001)
[INFO] [0/Radio_1] Initializing block control (NOC ID: 0x12AD100000000001)
[INFO] [0/DDC_0] Initializing block control (NOC ID: 0xDDC0000000000000)
[INFO] [0/DDC_1] Initializing block control (NOC ID: 0xDDC0000000000000)
[INFO] [0/DUC_0] Initializing block control (NOC ID: 0xD0C0000000000000)
[INFO] [0/DUC_1] Initializing block control (NOC ID: 0xD0C0000000000000)
[INFO] [MULTI_USRP]     1) catch time transition at pps edge
[INFO] [MULTI_USRP]     2) set times next pps (synchronously)
[WARNING] [STREAMER] Requested send_frame_size of 8000 exceeds the maximum possible on this stream. Using 7972
[WARNING] [STREAMER] Requested send_frame_size of 8000 exceeds the maximum possible on this stream. Using 7972
[WARNING] [STREAMER] Requested recv_frame_size of 8000 exceeds the maximum possible on this stream. Using 7972
[WARNING] [STREAMER] Requested recv_frame_size of 8000 exceeds the maximum possible on this stream. Using 7972
Cell pci=1, bw=20 MHz, dl_arfcn=368500 (n3), dl_freq=1842.5 MHz, dl_ssb_arfcn=366970, ul_freq=1747.5 MHz

==== gNodeB started ===
Type <t> to view trace
2024-02-20T19:42:59.248943 [FAPI    ] [W] [    34.5] Real-time failure in FAPI: Received late DL_TTI.request from slot 34.4
2024-02-20T19:42:59.248961 [FAPI    ] [W] [    34.5] Real-time failure in FAPI: Received late UL_TTI.request from slot 34.4
^CStopping ..
2024-02-20T19:43:17.056328 [Low-PHY#0] [W] Real-time failure in low-phy: PRACH request late for sector 0, slot 781.1 and start symbol 0.

**Comment by pgawlowicz** [comment]
Hi @chiefmeta any update on this issue? could you try with the newest code?
